decidim-budgets/app/models/decidim/budgets/order.rb

```diff
@@ -108,6 +108,7 @@ def can_checkout?
       # enabled
       def budget_percent
         return (total_projects.to_f / maximum_projects) * 100 if projects_rule?
+        return 0 if total_budget.zero?
```

Comment:
```suggestion
        return 0 if budget.total_budget.zero?
```

Reply:
if the local method `total_budget` is 0, then the below division would be mathematically valid. What we want to avoid here is having a division by 0. So please refer above to `budget.total_budget`